### DISCUSSION TITLE
Problem during creating a website

### DISCUSSION THREAD

**Discussion by biswa-the-great** [github/india#344] [General]

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
